djstripe/models.py

```diff
@@ -633,9 +650,16 @@ def plan(self):
 
 @class_doc_inherit
 class UpcomingInvoice(Invoice):
-    __doc__ = getattr(Invoice, "__doc__")
+    """A model extending Invoice providing dj-stripe functionality.
+
+    An UpcomingInvoice does not save to our database, it is obtained from
+    the Stripe api everytime.
+    """
+
+    __doc__ += getattr(Invoice, "__doc__")
 
     def __init__(self, *args, **kwargs):
+        """Assign _invoiceitems to an empty list."""
```

Comment:
Put something more generic or leave it out. This just describes line 664, which is very straightforward anyway